pkg/model/rhsso_deployment.go

```diff
@@ -239,7 +239,9 @@ func RHSSODeploymentReconciled(cr *v1alpha1.Keycloak, currentState *v13.Stateful
 	reconciled.Spec.Template.ObjectMeta.Annotations = AddPodAnnotations(cr, reconciled.Spec.Template.ObjectMeta.Annotations)
 
 	reconciled.ResourceVersion = currentState.ResourceVersion
-	reconciled.Spec.Replicas = SanitizeNumberOfReplicas(cr.Spec.Instances, false)
+	if cr.Spec.DisableReplicasSyncing != true {
+		reconciled.Spec.Replicas = SanitizeNumberOfReplicas(cr.Spec.Instances, false)
+	}
```

Comment:
Could you please do a similar implementation in `keycloak_deployment` as well?

Reply:
Absolutely! thank you